package.json

```diff
@@ -21,5 +21,8 @@
 	"devDependencies": {
 		"eslint": "^8.29.0",
 		"husky": "^8.0.3"
+	},
+	"dependencies": {
+		"@radix-ui/react-tabs": "^1.0.4"
```

Comment:
**question:** Shouldn't this be in the `/frontend/package.json`

Reply:
Right, must've run the install pkg cmd in the root directory